standalone/detectors/fasttrack/include/threadstate.h

```diff
@@ -1,32 +1,80 @@
 #ifndef THREADSTATE_H
-#define THREADSTATE_H
+#define THREADSTATE_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define THREADSTATE_H
```